include/mesh/exodusII_io.h

```diff
@@ -140,6 +140,23 @@ class ExodusII_IO : public MeshInput<MeshBase>,
    */
   void write_complex_magnitude (bool val);
 
+  /**
+   * If true this flag enforces the following behaviors:
+   *
+   * .) When reading an Exodus file, instead of setting the
+   *    elem_num_map and node_num_map ids directly on Nodes and Elems
+   *    read in from the Exodus file, set their unique_ids
+   *    instead. Normally libmesh chooses the unique_ids automatically
+   *    but in this case we override that choice.  As a consequence
+   *    the libMesh Elems/Nodes will be numbered based on their
+   *    implied ordering in the exo file (sequentially by block and
+   *    without any gaps in the numbering).
+   * .) When writing an Exodus file, populate the elem_num_map and
+   *    node_num_map with the unique_ids of the Elems/Nodes being
+   *    written.
```

Comment:
Let's add a comment on why one might want to do this?  (ReplicatedMesh using `O(max_id)` storage)